pkg/security/admission/admission_test.go

```diff
@@ -68,9 +68,11 @@ func TestAdmit(t *testing.T) {
 			Type: kapi.SupplementalGroupsStrategyMustRunAs,
 		},
 		Groups: []string{"system:serviceaccounts"},
+		// give this scc priority since it is what we want to validate with
+		SortPriority: 1,
```

Comment:
don't you need a more restrictive SCC that also matches to actually check that you're matching on sort priority, and not just the secondary `pointScore` sort?